File: samples/snippets/csharp/VS_Snippets_CLR_System/system.Char.GetNumericValue/CS/getnumericvalue.cs
// <snippet1>
using System;

public class GetNumericValueSample {
	public static void Main() {
		string str = "input: 1";

		Console.WriteLine(char.GetNumericValue('8'));		// Output: "8"
		Console.WriteLine(char.GetNumericValue(str, 7));	// Output: "1"
	}
}
// </snippet1>

File: snippets/fsharp/VS_Snippets_CLR_System/system.Char.GetNumericValue/FS/getnumericvalue.fs
module getnumericvalue1

// <snippet1>
open System

let str = "input: 1"

printfn $"{Char.GetNumericValue '8'}"       // Output: "8"
printfn $"{Char.GetNumericValue(str, 7)}"   // Output: "1"

// </snippet1>